tests/integration/server/users_api_tests.py

```diff
@@ -50,3 +51,21 @@ def test_invalid_duplicate_username():
         assert response.json()["message"] == expected["message"]
         assert response.json()["code"] == expected["code"]
         delete_user_by_username(user_input["username"])
+
+
+def test_delete_user_by_next_id():
```

Comment:
You should also include a bad test where you attempt to delete a user by using a next_id that does not exist.